macos/FluentUI/AvatarView.swift

```diff
@@ -60,7 +60,7 @@ open class AvatarView : NSView {
 	}
 	
 	@available(*, unavailable) required public init?(coder decoder: NSCoder) {
-		fatalError("init(coder:) has not been implemented")
+		preconditionFailure("init(coder:) has not been implemented")
```

Comment:
For all of this, imo the "init(coder:) has not been implemented" can be removed. As written, it makes it sound like we want to implement it but haven't. And having @available(*, unavailable) makes the intention quite clear so I don't think the text adds much

Reply:
We may want to revisit this in the future to unblock clients who use storyboards/xibs for our public components, but short term making the APIs unavailable should get the message across.